app/views/profile/show.html.erb

```diff
@@ -2,35 +2,37 @@
   <%= render 'users/header' %>
 
   <!--main content-->
-  <% unless @user.profile.title.blank? %>
-    <h3 class="heading"><%=h @user.profile.title %></h3>
+  <% if @profile.title.present? %>
+    <h3 class="heading"><%=h @profile.title %></h3>
   <% end %>
 
   <div class="wrapper">
     <dl class="meta">
-      <dt class="pseuds"><%= link_to ts("My pseuds:"), user_pseuds_path(@user) %></dt>
-      <dd class="pseuds"><%= print_pseud_list(@user.pseuds) %></dd>
+      <dt class="pseuds">
+        <%= link_to all_pseuds_link(@profile), user_pseuds_path(@user) %>
```

Comment:
I still think that the link should come inside the `dd`, not the `dt`. (Or in both, if you're set on having it in the `dt`.) Having the text `including X more` here, instead of after the list of pseuds, seems like it would be confusing. It also means that someone might think that the `dd` already includes the X more.

Reply:
@tickinginstant I am not set on anything! I am just tired and confused. Would you mind making a PR to my fork (or directly on my PR if technically possible!) with the changes you'd like to see regarding the `dt`/`dd`?

Reply:
The simple way would just be to have it look something like this:
```
<dt class="pseuds"><%= ts("My pseuds:") %></dt>
<dd class="pseuds">
  <p><%= print_pseud_list(@profile.pseuds) %></p>
  <p>(<%= link_to all_pseuds_link(@profile), user_pseuds_path(@user) %>)</p>
</dd>
```
Though I think my true preference would be for [something like this](https://github.com/tickinginstant/otwarchive/commit/9e68b3e825354c98e407428e6216ffd12ed6c576).

Reply:
+1 to @tickinginstant's simple way.

Reply:
Fixed in my PR to Cristina's